machAeroTutorials/overset_theory.rst

```diff
@@ -0,0 +1,158 @@
+.. _overset_theory:
+
+########################
+Overset Theory
+########################
+
+Overset Mesh
+=============
+
+ADFlow can only process multiblock structured meshes. For simple geometries, this is fine. 
+But it can be really hard to generate a single structured mesh for a complex geometry. 
+It might even be impossible to archive the required mesh quality.
+
+To mitigate this problem, the overset approach (also called chimera-patch) was developed. Instead 
+of having one big structured mesh, the fluid domain is split up in different, overlapping meshes. The 
+fluid solver then interpolates between those. Typically, there are one farfield and multiple nearfield 
+meshes:
+
+.. figure:: images/overset_Overview.png
+    :align: center 
+
+    Multiple nearfield and one farfield mesh can be seen.
+..
+    src: https://openmdao.org/wp-content/uploads/2018/06/bli_16_9_clean.png
+
+More about the overset implementation in ADFlow can be found here: `An Efficient Parallel Overset 
+Method for Aerodynamic Shape Optimization 
+<https://www.researchgate.net/publication/313459613_An_Efficient_Parallel_Overset_Method_for_Aerodynamic_Shape_Optimization>`_\.
+
+.. note:: As the solver has to interpolate in the overlapping region, the calculated solution will locally not be
+          as accurate. This means it should not happen in critical regions, like the wing tip.
+
```

Comment:
I would be a bit more explicit saying `It is recommended to avoid such overlapping near critical regions of the flowfield, like the wing tip`. Also, maybe change `calculated solution` into `numerical solution`?